src/components/Modal/index.js

```diff
@@ -1,17 +1,16 @@
+import { Elements, StripeProvider } from 'react-stripe-elements'
 import React, { useRef } from 'react'
-import { useStore, useActions } from 'easy-peasy'
-import { StripeProvider, Elements } from 'react-stripe-elements'
-import styled from 'styled-components'
-
-import useOnClickOutside from '../../hooks/useOnClickOutside'
-import useScript from '../../hooks/useScript'
+import { useActions, useStore } from 'easy-peasy'
 
-import Header from './Header'
+import Cart from '../Cart'
```

Comment:
@s0kil is this a plugin that has changed the order of the imports?

Can we keep absolutes up top and then relative imports below (with hooks/lib/utils above components). I'll better document this in contributing notes.

Reply:
Yes, Plugin auto sorts imports by name.